### PR TITLE
data(ozone)

### PR DESCRIPTION
**Pull request by lucasrodes**
Ads the dataset on the consumption of controlled substances. Currently, the metrics from this dataset are present: https://owid.cloud/admin/datasets/2439, which was manually created years ago from different datasets.

This PR adds all the ETL steps. 

Heads up on the snapshot step: there are multiple source files; their DVCs should be equivalent.

related issue: https://github.com/owid/owid-issues/issues/959

TODOs:

- [x] Solve issues raised in https://github.com/owid/owid-issues/issues/959#issuecomment-1476750410
- [x] Add dataset description

### CODE REVIEW

File: etl/steps/data/garden/unep/2023-03-17/consumption_controlled_substances.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+def add_regions(df: pd.DataFrame) -> pd.DataFrame:
+    id_vars = ["country", "year"]
+    var_name = "chemical"
+    value_name = "consumption"
+    # Pivot
+    df_pivot = df.pivot(index=id_vars, columns=[var_name], values=value_name).reset_index()
+    # Add region data
+    regions = ["World", "Asia", "Africa", "North America", "South America", "Oceania"]
```
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/unep/2023-03-17/consumption_controlled_substances.py:85]: Also would it not make sense to include Europe in here too? As the continent as well as the European Union (28).

**Review by spoonerf — CHANGES_REQUESTED** [review]
Hey @lucasrodes - just a couple of issues but mostly looks good. Thanks for the opportunity to review - I always learn a lot from your code.

File: etl/steps/data/garden/unep/2023-03-17/consumption_controlled_substances.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/unep/2023-03-17/consumption_controlled_substances.py]: Don't some of the existing chart use global totals - or do you mean you will calculate it somewhere else?

[the remark this answers is not in the document]
Comment (lucasrodes) [reply etl/steps/data/garden/unep/2023-03-17/consumption_controlled_substances.py]: I meant that the default `add_region_aggregates` function is not helpful here to estimate the data for Europe, as data for all EU28 member states is missing (UNEP just shares data at European Union level).

However, we can still estimate the data for Europe by adding `European Union (28) + Albania + ...`.

**Review by spoonerf — APPROVED** [review]

**Pull request merged**